appveyor.yml

```diff
@@ -40,6 +40,9 @@ platform:
     - x64
 
 install:
+    # Skip if we are not in a PR.
+    - cmd: if not defined APPVEYOR_PULL_REQUEST_NUMBER exit 0
```

Comment:
Looks like the [answer]( http://help.appveyor.com/discussions/questions/1981-test_script-in-powershell-how-to-exit-when-success-or-when-fail ) is we can't with a success status. So, it must be exit 1. That's a bit ugly.